app/views/people/_relationship_action.haml

```diff
@@ -1,7 +1,8 @@
 - unless person == current_user.person
   - contact = current_user.contacts.find_by_person_id(person.id)
   - contact ||= Contact.new(:person => person)
-  = aspect_membership_dropdown(contact, person, 'right')
+  - gon_load_contact(contact)
+  .aspect_membership_dropdown.placeholder{data: {person_id: contact.person_id}}
```

Comment:
Is this partial used too diversely for that logic to move to a controller? Refactoring it into a view helper would be great anyhow I guess.